condor/condor_meter

```diff
@@ -561,7 +561,7 @@ def determine_host_description(classad):
         return None
 
     # Check first for SE-based matching.
-    if ('DESIRED_SEs' not in classad) or ('MATCH_GLIDEIN_SEs' not in classad):
+    if not classad.get('DESIRED_SEs') or classad.get('MATCH_GLIDEIN_SEs'):
```

Comment:
I'm gonna switch this to check against the `None` value - but I think order of operations wise this change is missing a `not` on the right hand of the `or`...